packages/storybook/stories/Stateful Client/Adapters.stories.mdx

```diff
@@ -0,0 +1,136 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import { Canvas, Description, Heading, Props, Source, Title } from '@storybook/addon-docs/blocks';
+
+<Meta id="statefulclient-reacthooks-setup" title="Stateful Client/React Hooks/Adapters" />
+
+# Adapters for Composites
```

Comment:
Should we also mention somewhere in this section that developers can create their own custom adapters and aren't forced into using the provided adapters?

Reply:
Lets make this addition later but its a good area we can explore in future updates

Reply:
I think when we are ready to talk about custom adapters lets add in that content then.